notifications/front_end/src/components/NavBarComp.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import { Navbar, Nav, Container, Button } from "react-bootstrap";
+import logo_white from '../assets/logo_white.png'
+import { Link } from "react-router-dom";
+import { FaUserCircle } from "react-icons/fa";
+import { useNavigate } from "react-router-dom";
+
+ const NavBarComp = ({authenticate}) => {
+  const navigate = useNavigate();
+  const signOut = () => {
+    authenticate();
+    navigate("/");
+    
+    
+};
+   
+  return (
```

Comment:
Is this supposed to be in NavBarComp? Or does the export statement execute this?

Reply:
The export grabs whatever is in the return and spits it out (I think that's what you're asking)